include/sys/time_units.h

```diff
@@ -66,6 +76,7 @@ static ALWAYS_INLINE u64_t z_tmcvt(u64_t t, u32_t from_hz, u32_t to_hz,
 	}
 
 	u64_t off = 0;
+	u64_t result = 0;
```

Comment:
Whoops, I forgot I had made that change and did a build assuming it was fixed on C++11. It appears to be a giant pain to make this work in C++11 because += is prohibited, and it's been 6 years since 2014 anyway, so I dropped the change to this function and fixed the commit message.